### DISCUSSION TITLE
Customizing Bootstrap JS plugins

### DISCUSSION THREAD

**Discussion by pratham2003** [holger1411/understrap#1288] [General]
Is there a way to customize which Bootstrap JS plugins we want to include in the main JS file?

It would be nice to be able to selectively import or even dynamically import javascript modules on demand during runtime.
Bootstrap 4 has this: https://getbootstrap.com/docs/4.0/getting-started/webpack/

Can't we just use the above and replace the way bootstrap is currently integrated?
What are the possible challenges?

**Comment by axlright** [comment]
See Issue #1116 

I'll post a workflow shortly as I'm working on reducing js bloat via the current system until there's a more comprehensive solution.

**Comment by bacoords** [comment]
This is definitely something I'd recommend doing in a [child theme](https://github.com/understrap/understrap-child). I've done it before (by rewriting the gulpfile and enqueueing each separately) and would be happy to provide a gist if that's still wanted.

**Comment by danemorgan** [comment]
@bacoords is this something we could include in our child theme documentation?

**Comment by 0dp** [comment]
Isn't this the same as #1287

**Comment by Thomas-A-Reinert** [comment]
It absolutely is, @0dp
